client.go

```diff
@@ -405,7 +410,7 @@ func unixDial(networ, addr string) (net.Conn, error) {
 		if err != nil {
 			return nil, fmt.Errorf(gettext.Gettext("cannot resolve unix socket address: %v"), err)
 		}
-	} else {
+	} else { // TODO - I think this is dead code
```

Comment:
Really? Isn't that code path required for any socket which isn't in /var/lib/lxd ?

The other codepath has an hardcoded socket path, so if the else is never used, we've got another problem :)